### PR TITLE
Google Drive Add

### PR DESCRIPTION
**Pull request by Himusaki**
Can u add me to your google Shared drive ?
farukimatona@gmail.com

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ Lorem ipsum i am too lazy figure what it does yourself
 
 You might be lazy too so here ya go:
 
-[![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy?template=https://github.com/patheticGeek/torrent-aio-bot)
+[![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy?template=https://github.com/sobuj53/torrent-aio-bot)
```
Comment (patheticGeek) [inline Readme.md:7]: This commit doesn't need to be in this merge

File: lib/bot.js [hunk 2]
```diff
@@ -167,20 +167,20 @@ function bot(torrent, bot) {
...
-        mess1 += `Speed: ${torr.speed}\n\n`;
-        mess1 += `Progress: ${torr.progress}%\n\n`;
-        mess1 += `Time Remaining: ${torr.redableTimeRemaining}\n\n`;
+        mess1 += `Downloaded ✅: ${torr.downloaded}\n\n`;
+        mess1 += `Speed 🚀: ${torr.speed}\n\n`;
+        mess1 += `Progress 📥: ${torr.progress}%\n\n`;
+        mess1 += `Time Remaining ⏳: ${torr.redableTimeRemaining}\n\n`;
       } else {
-        mess1 += `Link: ${torr.downloadLink}\n\n`;
+        //mess1 += `Download Link 🔗: ${torr.downloadLink}\n\n`;
```
Comment (patheticGeek) [inline lib/bot.js:179]: Keep the direct link for if the user wants to stream

**Review by patheticGeek — CHANGES_REQUESTED** [review]
This is a good PR, just do the two changes and I'll merge.
Appreciate the time you put in :+1:

**Comment by patheticGeek** [comment]
> Can u add me to your google Shared drive ?
> [farukimatona@gmail.com](mailto:farukimatona@gmail.com)

My shared drive is fked, not using it anymore as downloads and streams are blocked

**Comment by sobuj53** [comment]
I think the user must have created this pr by mistake. My repo is almost in full sync with your reop. Those are just few changes that are very custom to my likings. In short you can close this pr. Thank you for your awesome repo.

**Pull request closed without merging**